app/assets/stylesheets/views/_travel-advice.scss

```diff
@@ -1,253 +1,88 @@
-            position: absolute;
-            left: -9999em;
-          }
-
-          input {
-            display: block;
-            border-color: #A6AAAC;
-            margin: 0;
-          }
-        }
-      }
+  .app-full-width-break {
+    clear: both;
+    margin-right: govuk-spacing(3);
+    margin-left: govuk-spacing(3);
   }
 
-  .country-metadata {
-    padding: 0;
-    margin: 0;
-    color: $govuk-secondary-text-colour;
+  .countries-list {
+    border-bottom: 1px solid $govuk-border-colour;
+    display: block;
+    padding-bottom: govuk-spacing(4);
 
     li {
-      @include govuk-font(16);
-
-      list-style: none;
-      padding: 0;
-      .label {
-        display: inline-block;
-        width: 7em;
-      }
-      p {
-        @include govuk-font(16);
-      }
+      box-sizing: border-box;
+      color: $govuk-secondary-text-colour;
+      float: left;
+      padding-right: govuk-spacing(3);
+      width: 50%;
     }
   }
 
-  .meta-data .print-link {
-    margin: 1em 0;
+  .countries-list__link {
+    margin-right: govuk-spacing(2);
   }
 
-
- .subscriptions  {
-
-    @include govuk-media-query($from: desktop) {
-      ul li:first-child {
-        margin-right: 0;
-        margin-left: 0;
-      }
-    }
-
-    p {
-      display: inline-block;
-      margin-right: 0.5em;
-      font-weight: 700;
-    }
-
-    @include media-down(tablet) {
-      text-align: left;
-      width: auto;
-      padding-top: 0;
-
-
+  .countries-wrapper {
+    h2 {
+      margin: 0;
     }
 
-    ul {
-      list-style: none;
+    .country-count {
       margin: 0;
-      padding: 0;
-      display: inline-block;
 
-      li {
-        margin-bottom: 0.25em;
-        text-align: left;
-        display: inline;
-      }
-
-      li:first-child {
-        margin-right:1.5em;
+      .filter-count {
+        @include govuk-font(80, $weight: bold);
       }
+    }
 
-      a {
-        text-decoration: none;
-        font-weight: 700;
-        padding-left: 28px;
-        height: 16px;
-        background-position: 0 50%;
-        background-repeat: no-repeat;
-
-        &.feed {
-          background-image: image-url("travel-advice/feed-icon-black.png");
-        }
-
-        &.email-alerts {
-          background-image: image-url("travel-advice/mail-icon.png");
-          @include govuk-device-pixel-ratio() {
-            background-image: image-url("travel-advice/mail-icon-x2.png");
-            background-size: 20px 14px;
-          }
-        }
-      }
+    h3 {
+      @include govuk-font(80, $weight: bold);
+      margin-top: 0.37em;
+      margin-bottom: 0;
+      padding-top: 0;
     }
   }
+}
 
-  .page-navigation {
-    border-bottom: 1px solid $govuk-border-colour;
-    margin: 0 0 2.5em;
-    padding-bottom: 1em;
-  }
+// TODO: This #wrapper.travel-advice turns off the default margin and width,
+// which now isn't needed thanks to the grid in GOV.UK Frontend.
+#wrapper.travel-advice {
+  margin: 0;
+  max-width: none;
```

Comment:
Can you explain why this is here? I had a quick look and turning off these styles doesn't seem to make any difference, not sure the comment is clear enough.

Reply:
Good catch! It's a hangover from a previous step on this branch that I forgot to remove - gone now.